website/templates/public/register.mako

```diff
@@ -141,6 +141,14 @@
                         </div>
                     </div>
                 </div>
+                <!-- Terms of Service and Privacy Policy agreement -->
+                <div class="form-group">
+                    <div style="float: right">
```

Comment:
Minor: I think you can/should use `.pull-right` here rather than use inline styling.